HW2/src/test/java/CheckCreatedBoard.java

```diff
@@ -0,0 +1,109 @@
+import dto.BoardDto;
+import dto.ListDto;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import service.RestBoardAssertions;
+import service.RestBoardService;
+import service.RestListAssertions;
+import service.RestListService;
+
+public class CheckCreatedBoard {
+
+    @BeforeMethod
+    void beforeMethod() {
+
+        BoardDto[] allBoards = new RestBoardService()
```

Comment:
why u delete all boards? it could affect test data

Reply:
Because I understand that before each test  Trello have to be empty